frameworks/elastic/tests/test_shakedown.py

```diff
@@ -6,13 +8,15 @@
 import sdk_upgrade
 from tests.config import *
 
+log = logging.getLogger(__name__)
+
 FOLDERED_SERVICE_NAME = sdk_utils.get_foldered_name(PACKAGE_NAME)
```

Comment:
Yes, it's difficult to find a balance. I've started removing the `*` imports when I touch a file, but since this PR is already pretty big we can address that in a later one here.